app/views/publishers/organisations/show.html.slim

```diff
@@ -15,13 +15,12 @@
 
     = render "organisation", organisation: @organisation
 
-    - if @organisation.school_group?
-
+    - if current_organisation.trust?
       h2.govuk-heading-m class="govuk-!-margin-top-8"
         = t(".schools")
 
       = govuk_summary_list do |summary_list|
-        - @organisation.schools.not_closed.order(:name).partition(&:profile_complete?).reverse.flatten.each do |school|
+        - current_organisation.schools.not_closed.order(:name).partition(&:profile_complete?).reverse.flatten.each do |school|
```

Comment:
seems like complicated ruby to have in a view but realise was there already, so your call if you do anything about it!

Reply:
I'll have a think and discuss with people and if we want to move this into the controller or something we can do that in a later PR I guess!